### ISSUE TITLE
Extra static files created by `hugo serve`

### ISSUE THREAD

**Issue by yurishkuro** [jaegertracing/documentation#153]
When I run `hugo server`  (or `make develop`) on a clean master, it always generates two files, like these:
```
resources/_gen/assets/sass/sass/style.sass_d0f3690595a16dc8f17d99e9a9142337.content
resources/_gen/assets/sass/sass/style.sass_d0f3690595a16dc8f17d99e9a9142337.json
```
Seems like we should .gitignore things under `_gen`, but we also have two files checked in there.

@lucperkins  Is this because of netlify issue with hugo installation?

**Comment by lucperkins** [comment]
@yurishkuro Yes, this is unfortunately something we’ll just have to deal with for now. Basically Netlify doesn’t yet let you use the “extended” version of Hugo with built-in Sass support. Checking the resources folder into Git is the currently available workaround for that. Before committing things to Git, make sure to delete that folder and then re-build, which will always produce the most up-to-date resources folder.

I’ve asked Netlify several times about this and have yet to hear back. Alas, it’s either this or using a Node.js-based Sass pipeline.

**Comment by caspark** [comment]
@lucperkins it seems like these days Netlify does use Hugo extended; from my deploy logs:

```
10:47:45 PM: Installing Hugo 0.54.0
10:47:47 PM: Hugo Static Site Generator v0.54.0-B1A82C61A/extended linux/amd64 BuildDate: 2019-02-01T10:04:38Z
```

Does that mean these resources can be safely gitignored, and Hugo adjusted to stop generating them?

**Comment by lucperkins** [comment]
@caspark This has actually already been addressed. Just forgot to close the issue 🙃

**Comment by yurishkuro** [comment]
Fixed by #201
